.github/workflows/release-published.yml

```diff
@@ -63,243 +78,243 @@ jobs:
         env:
           GPG_PASSWORD: ${{ secrets.GPG_PASSPHRASE }}
 
-      - name: Publish to Maven Central
-        env:
-          MAVEN_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
-          MAVEN_PASSWORD: ${{ secrets.SONATYPE_TOKEN }}
-          GPG_PASSWORD: ${{ secrets.GPG_PASSPHRASE }}
-        run: |
-          version=${{ needs.setup.outputs.version }}
+  #     - name: Publish to Maven Central
```

Comment:
These shouldn't be commented out?

Reply:
this is just for testing purposes to see if we get the point after Manual deployment